paddle/fluid/operators/controlflow/conditional_block_op.cc

```diff
@@ -295,6 +295,35 @@ class ConditionalBlockGradOp : public ConditionalOp {
   }
 };
 
+template <class T>
+struct FilterUnsupportDtype {};
```

Comment:
```suggestion
struct FilterNoGradInput {};
```

另外这里的是个空struct? 为什么给一个默认 static void filter实现？里面什么都不做，直接返回 vec，这样下面的GradMaker里就可以用 FilterNoGradInput<T>::filter() 了